docs: add documentation around the new `x-codeSamples` spec extension (#81)
bump-sh/docs: docs/help/specifications-support/openapi-support.md

JSON Schema provides the possibility to declare a property as read or write only. Read more in [the JSON Schema section of this documentation](/specifications-support/json-schema.md#readonly-and-writeonly-properties).

## Specification extensions

> These are additional properties which are not specificed by the OpenAPI specification but useful for you to be able to customize your documentation content. All those properties start with the `x-` naming convention to be identified as “eXternal” from the OpenAPI specification.

Reviewer: ```suggestion
These are additional properties that are not specified by the OpenAPI specification but are helpful for you to customize your documentation content. All those properties start with the `x-` naming convention to be identified as “eXternal” from the OpenAPI specification.
```
